fern/definition/medical/document.yml

```diff
@@ -79,25 +87,18 @@ service:
           patientId:
             docs: The ID of the Patient for which to list available Documents.
             type: string
-          facilityId:
-            docs: The ID of the Facility where the patient is receiving care.
-            type: string
           dateFrom:
             docs: The start date (inclusive) for which to filter returned documents - formatted `YYYY-MM-DD` as per ISO 8601. If not provided, no start date filter will be applied.
             type: optional<string>
           dateTo:
             docs: The end date (inclusive) for which to filter returned documents - formatted `YYYY-MM-DD` as per ISO 8601. If not provided, no end date filter will be applied.
             type: optional<string>
+          content:
+            docs: |
+              Value to search on the document reference (minimum 3 chars).
```

Comment:
I know this is part of the docs, but the description doesn't make sense - we're not only searching the contents of the DocRef, but the contents of the document itself as well. Let's update the description accordingly in both places.

Reply:
Replacing with:

"Value to search within the document reference and the actual contents of the document (minimum 3 chars)."